File: PiwikTracker/PTEventEntity.m
//
//  PTEventEntity.m
//  PiwikTracker
//
//  Created by Mattias Levin on 8/6/13.
//  Copyright (c) 2013 Mattias Levin. All rights reserved.
//

#import "PTEventEntity.h"


@implementation PTEventEntity

@dynamic date;
@dynamic requestParameters;


- (void)awakeFromInsert {
  // Set date
  self.date = [NSDate date];
}


@end

File: PiwikTracker-Swift/PTEventEntity.swift
//
//  PTEventEntity.swift
//  PiwikTracker
//
//  Created by Cornelius Horstmann on 07.10.16.
//  Copyright © 2016 Mattias Levin. All rights reserved.
//

import Foundation
import CoreData

internal class PTEventEntity: NSManagedObject {
    public var date: Date?
    public var piwikRequestParameters: NSData?
    
    override func awakeFromInsert() {
        date = Date()
    }
}
